crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -716,6 +716,7 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                     $$"""
                     // <auto-generated />
                     #nullable enable
+                    #pragma warning disable CS8981
```

Comment:
Do we really need to do this in _all_ generated code? I'd say let's disable it in our own test projects, user projects usually shouldn't need it.

Reply:
Ah there could be points made for both directions;

ie reducers being callable from the cmdline there's a point to name them in snake_case, and it fits the naming convention of tables too.

But you're also right it opens the door for non-idiomatic C# names to make it through. I'm fine with either solutions.